conf4j-core/src/main/java/org/conf4j/core/source/reload/ReloadStrategy.java

```diff
@@ -0,0 +1,8 @@
+package org.conf4j.core.source.reload;
+
+public interface ReloadStrategy {
```

Comment:
Something doesn't click here. 
1. Name. The design pattern Strategy in Wikipedia says:  

> is a behavioral software design pattern that enables selecting an algorithm at runtime. Instead of implementing a single algorithm directly, code receives run-time instructions as to which in a family of algorithms to use.[1]

So the name suggest this strategy will now how to reload something (we don't know what). I would have expected it to have a method like reload() returning X.
Having start() and stop() sounds bizarre, and not an algorithm or a specific action. Also, start() consumes an abstract reloadCallback which makes it very hard to grasp what this interface function is all about.